lib/core/test/unit/Cardano/Wallet/Primitive/CoinSelection/BalanceSpec.hs

```diff
@@ -679,13 +658,13 @@ prop_performSelection_small mockConstraints (Blind (Small params)) =
         "not nonZeroExtraCoinSource && not nonZeroExtraCoinSink" $
 
     -- Inspect the sets of minted and burned assets:
-    cover 20 (view #assetsToMint params /= TokenMap.empty)
+    cover 20 (view #assetsToMint params /= mempty)
```

Comment:
I feel this is one of those cases where indicating the type helps with readability.

In particular, it's useful to distinguish between `TokenMap` and `TokenBundle`, as only the latter type allows ada quantities.

Perhaps we can use a type annotation? (`mempty :: TokenMap`).